boa/src/value/mod.rs

```diff
@@ -40,6 +42,16 @@ pub use hash::*;
 pub use operations::*;
 pub use r#type::Type;
 
+static BIG_INT_2_E_64: Lazy<BigInt> = Lazy::new(|| {
+    const TWO_E_64: u128 = 2u128.pow(64);
+    BigInt::from(TWO_E_64)
+});
+
+static BIG_INT_2_E_63: Lazy<BigInt> = Lazy::new(|| {
+    const TWO_E_63: u128 = 2u128.pow(63);
+    BigInt::from(TWO_E_63)
+});
+
```

Comment:
I think we can skip the `BIG_INT` prefix. It already is a BigInt type, so `2_E_{N}` should be enough.